recipes/hiredis/0.14.1/conanfile.py

```diff
@@ -0,0 +1,78 @@
+from conans import AutoToolsBuildEnvironment, ConanFile, tools
+
+from os import rename
+from os.path import join
+
+
+class HiredisConan(ConanFile):
+    name = "hiredis"
+
+    description = "Minimalistic C client for Redis >= 1.2"
+    topics = "conan", "c", "redis"
+
+    license = "BSD 3-Clause \"New\" or \"Revised\" License"
```

Comment:
```suggestion
    license = "BSD-3-Clause"
```

Reply:
In the next version cmake will be used as build system, therefore the current version will be something special and I thought a separate folder for this version would fit it best.